cgroup-limit/test.sh

```diff
@@ -34,7 +34,10 @@ SYSTEMD_RUN="systemd-run"
 if [ "$UID" != "0" ]; then
   if ! grep -q "cpu" "/sys/fs/cgroup/user.slice/user-$UID.slice/user@$UID.service/cgroup.controllers" ; then
     # user can't set cpu limits, use sudo.
-    SYSTEMD_RUN="sudo -n $SYSTEMD_RUN -E PATH=$PATH -E DOTNET_ROOT=$DOTNET_ROOT"
+    SYSTEMD_RUN="sudo -n $SYSTEMD_RUN"
+
+    # Pass DOTNET_ROOT to support testing against a dotnet tarball.
+    [ -z "${DOTNET_ROOT}" ] || SYSTEMD_RUN="$SYSTEMD_RUN -E DOTNET_ROOT=$DOTNET_ROOT"
```

Comment:
I am not sure I follow this change.

`DOTNET_ROOT` is defined on RHEL 7 when we are testing our packages. It's set here: https://git.centos.org/rpms/rh-dotnet60/blob/c7-dotnet/f/SPECS/rh-dotnet60.spec#_66. So that still results in us trying to run `systemd-run -E` which fails as expected.

Reply:
> DOTNET_ROOT is defined on RHEL 7 when we are testing our packages.

Can you remind me why we're setting `DOTNET_ROOT`? Don't we have the root in `/etc/dotnet/install_location`?

Reply:
Probably because of the scl.

I'll change the test to not add the arg on rhel7.